src/lib/ArticleList/ArticlePreview.svelte

```diff
@@ -55,7 +54,7 @@
 		<span>Read more...</span>
 		<ul class="tag-list">
 			{#each article.tagList as tag}
-				<li class="tag-default tag-pill tag-outline"><a href="/?tag={tag}">{tag}</a></li>
+				<li class="tag-default tag-pill tag-outline">{tag}</li>
```

Comment:
this now seems to match what is on https://www.realworld.how/specifications/frontend/templates/, so I suppose it is fine